examples/01-filter/image-fft.py

```diff
@@ -0,0 +1,109 @@
+"""
+.. _image_fft_example:
+
+Fast Fourier Transform
+~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how to apply a Fast Fourier Transform (FFT) to a
+:class:`pyvista.UniformGrid` using :func:`pyvista.UniformGridFilters.fft`
+filter.
+
+Here, we demonstrate FFT usage by denoising an image, effectively removing any
+"high frequency" content by performing a `low pass filter
+<https://en.wikipedia.org/wiki/Low-pass_filter>`_.
+
+This example was inspired by `Image denoising by FFT
+<https://scipy-lectures.org/intro/scipy/auto_examples/solutions/plot_fft_image_denoise.html>`_.
+
+"""
+
+import numpy as np
+
+import pyvista as pv
+from pyvista import examples
+
+###############################################################################
+# Load the example Moon landing image and plot it.
+
+image = examples.download_moonlanding_image()
+print(image.point_data)
+
+# Create a theme that we can reuse when plotting the image
+grey_theme = pv.themes.DocumentTheme()
+grey_theme.cmap = 'gray'
+grey_theme.show_scalar_bar = False
+grey_theme.axes.show = False
+image.plot(theme=grey_theme, cpos='xy', text='Unprocessed Moon Landing Image')
+
+
+###############################################################################
+# Apply FFT to the image
+# ~~~~~~~~~~~~~~~~~~~~~~
+# FFT will be applied to the active scalars, ``'PNGImage'``, the default
+# scalars name when loading a PNG image.
+#
+# The output from the filter is a complex array stored by the same name unless
+# specified using ``output_scalars_name``.
+
+fft_image = image.fft()
+fft_image.point_data
+
+
+###############################################################################
+# Plot the FFT of the image
+# ~~~~~~~~~~~~~~~~~~~~~~~~~
+# Plot the absolute value of the FFT of the image.
+#
+# Note that we are effectively viewing the "frequency" of the data in this
+# image, where the four corners contain the low frequency content of the image,
+# and the middle is the high frequency content of the image.
+
+fft_image.plot(
+    scalars=np.abs(fft_image.point_data['PNGImage']),
+    cpos="xy",
+    theme=grey_theme,
+    log_scale=True,
+    text='Moon Landing Image FFT',
+)
+
+
+###############################################################################
+# Remove the noise from the ``fft_image``
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Effectively, we want to remove high frequency (noisy) data from our image.
+# First, let's reshape the size of the image.
```

Comment:
I think what this sentence means is "reshape [the 1d scalars array] to the size of the image", which is what happens later. In this context the original "by the size" is correct, because it's not the size of the image that gets reshaped.
```suggestion
# First, let's reshape by the size of the image.
```